### PR TITLE
Fixed GUI Guess

### PR DESCRIPTION
**Pull request by gabe2762**
Added the longer than 15 letter constraint, and added the case where the user is told a correct guessed word was already guessed.

### CODE REVIEW

**Review by auanmu — CHANGES_REQUESTED** [review]
Since I’m on mobile hard for me to pick the exact line since it squishes the text. But the windows version you removed some extra stuff that I think needs to be there still. Compare what’s after the else statement in max and windows, it shows you getting rid of clear the input box and updating the rank in the windows version but the Mac file still has it. Should be the same, need keep those lines of code.

File: MVC/view/austinguitest.py [hunk 1]
```diff
@@ -170,22 +170,21 @@ def backspace(self):
     '''
     def makeGuess(self, *args):
         input = self.e.get()
-        #made it so userGuess returns true/false so that way we only insert valid words into the listbox.
-        if self.controller.checkInput(input.lower(), self.reqLetter.lower()) == True:
+        # Check if the input is within the required length range
+        if len(input) < 4 or len(input) > 15:
```
Comment (tmhughes1) [inline MVC/view/austinguitest.py:174]: I think this covers all cases, but I'll have to verify after testing it

File: MVC/view/austinguitest.py [hunk 2]
```diff
@@ -170,22 +170,21 @@ def backspace(self):
...
+        elif self.controller.checkInput(input.lower(), self.reqLetter.lower()) == True:
+            # Check if the guess is correct
             if self.controller.controllerUserGuess(input) == True:
                 self.listBox.insert(tk.END,input.lower())
             else:
-                messagebox.showinfo("Invalid Guess", "Please re-enter a guess.")
+                messagebox.showinfo("Invalid Guess", "Word is not in list.")
         else:
-            messagebox.showinfo("Invalid input", "Ensure each guess uses the required letter, and consists of letters only.")
-        #update the points and rank after every guess.
```
Comment (tmhughes1) [inline MVC/view/austinguitest.py:181]: Why did you remove this part? The makeGuess function needs to still update points, rank, and clear the input box.

**Review by tmhughes1 — CHANGES_REQUESTED** [review]
Some code got removed from austinguitest in makeGuess that shouldn't have been! Please fix.

**Review by tmhughes1 — APPROVED** [review]
Thanks for fixing!

**Review by auanmu — APPROVED** [review]
Thanks for fixing it, great work Gabe!

**Pull request merged**